scheduler/scheduler.go

```diff
@@ -848,7 +838,7 @@ func (s *scheduler) walkWorkflow(prnodes []*processNode, pbnodes []*publishNode,
 		} else {
 			depGroupMap[pr.Target] = depGroup{Plugins: []core.SubscribedPlugin{pr}}
 		}
-		s.walkWorkflow(pr.ProcessNodes, pr.PublishNodes, depGroupMap)
+		getWorkflowPlugins(pr.ProcessNodes, pr.PublishNodes, depGroupMap)
```

Comment:
Seems like descendent nodes can be appended to dg.Plugins multiple times, can it cause any problems?